packages/sdk-core/.gitignore

```diff
@@ -4,3 +4,5 @@ coverage
 /src/abi
 /src/typechain
 *.generated.ts
+/typechain-types
```

Comment:
We were including these before?

Reply:
no `typechain-types` is generated from the `.sol` file when you run `hardhat compile`, the other `/src/typechain` folder is generated from abi files which we pull from `ethereum-contracts` as part of the build process